woocommerce/woocommerce
# Add `store_id` to system_status REST API

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

This PR adds a unique `store_id` (added by: https://github.com/woocommerce/woocommerce/pull/40705) to the `WC_REST_System_Status_V2_Controller`. 



### How to test the changes in this Pull Request:



1. Start an environment via Jurassic Ninja using this branch (`add/store-id-to-telemetry-endpoint`) 
2. Go through the store setup process.
3. Create REST API keys ([as described here](https://woocommerce.github.io/woocommerce-rest-api-docs/#authentication))
4. Make a request to the `system_status` API (replace `my_site` with the domain of your Jurassic Ninja instance and `consumer_key` and `consumer_secret` from the above step):

```shell
$ curl https://my_site/wp-json/wc/v3/system_status -u consumer_key:consumer_secret
```

Check the JSON output for the `store_id`, for instance by using [`jq`](https://jqlang.github.io/jq/):

```shell
$ curl https://my_site/wp-json/wc/v3/system_status -u consumer_key:consumer_secret | jq .environment.store_id
```

### Changelog entry




<details>

#### Significance




#### Type




#### Message Includes the store_id added by #40705 to the system_status REST API.

#### Comment 

</details>
